schemas/com.snowplowanalytics.datamodeling/config/jsonschema/1-0-0

```diff
@@ -0,0 +1,115 @@
+{
+        "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+        "self": {
+                "vendor": "com.snowplowanalytics.datamodeling",
+                "name": "config",
+                "version": "1-0-0",
+                "format": "jsonschema"
+        },
+        "type": "object",
+        "properties": {
+                "schema": {
```

Comment:
Schema and data shouldn't explicitly be in the schema. They're the 'self-describing' part. The schema describes only the properties of the `data` part of the Self-describing object.

('schema' is in fact encapsulated by the 'self' portion of the schema itself.